tests/model_registry/conftest.py

```diff
@@ -535,236 +536,220 @@ def model_registry_pod(admin_client: DynamicClient, model_registry_namespace: st
 
 
 # =============================================================================
-# DUPLICATED RESOURCE FIXTURES
+# RESOURCE FIXTURES PARMETRIZED
 # =============================================================================
-
-
-@pytest.fixture(scope="class")
-def db_name_1() -> str:
-    """Generate a unique name for the first DB instance."""
-    return f"db-instance-1-{str(uuid.uuid4())[:8]}"
-
-
 @pytest.fixture(scope="class")
-def db_secret_1(model_registry_namespace: str, db_name_1: str, teardown_resources: bool) -> Generator[Secret, Any, Any]:
-    """Create the first DB secret."""
-    with Secret(
-        name=db_name_1,
-        namespace=model_registry_namespace,
-        string_data=MODEL_REGISTRY_DB_SECRET_STR_DATA,
-        label=get_model_registry_db_label_dict(db_resource_name=db_name_1),
-        annotations=MODEL_REGISTRY_DB_SECRET_ANNOTATIONS,
-        teardown=teardown_resources,
-    ) as secret:
-        yield secret
-
+def updated_dsc_component_state_parametrized(
+    request: FixtureRequest,
+    admin_client: DynamicClient,
+    dsc_resource: DataScienceCluster,
+    teardown_resources: bool,
+) -> Generator[DataScienceCluster, Any, Any]:
+    """Configure DSC to use parametrized Model Registry namespace"""
+    if not teardown_resources:
+        yield dsc_resource
+        return
 
-@pytest.fixture(scope="class")
-def db_pvc_1(
-    model_registry_namespace: str, db_name_1: str, teardown_resources: bool
-) -> Generator[PersistentVolumeClaim, Any, Any]:
-    """Create the first DB PVC."""
-    with PersistentVolumeClaim(
-        name=db_name_1,
-        namespace=model_registry_namespace,
-        accessmodes="ReadWriteOnce",
-        size="5Gi",
-        label=get_model_registry_db_label_dict(db_resource_name=db_name_1),
-        teardown=teardown_resources,
-    ) as pvc:
-        yield pvc
+    # Get the namespace name from the parameter
+    namespace_name = request.param["ns_name"]
+    original_components = dsc_resource.instance.spec.components
 
+    # First, remove the Model Registry to allow namespace changes
+    remove_patch = {
+        DscComponents.MODELREGISTRY: {
+            "managementState": DscComponents.ManagementState.REMOVED,
+        },
+    }
```

Comment:
The only concern I have left for the PR is this remove patch, other than that LGTM

Reply:
Is MR enabled by default? I don't understand this part.

Reply:
yup, removing it